src/queries/retry.ts

```diff
@@ -17,6 +17,7 @@ export const retryGraphqlResponseError = async <T>(
   try {
     return await query()
   } catch (error) {
+    core.warning(`[Before retry check] ${error}`) // TODO: Remove after finishing debug
```

Comment:
if `TypeError('test')` is thrown,  `[Before retry check] TypeError: test` is output! (no stacktrace)